binnacle/lib/screens/map_screen.dart

```diff
@@ -108,13 +124,11 @@ class _MapState extends State<MapScreen> {
     }
 
     if (_markers.length > MAX_MARKERS) {
-      // TODO: Use the navigator sliding window
-      // BUG: This doesn't work
```

Comment:
Does it work now?

Reply:
I fixed getting the context so the snackbar shows it instead of showing the big navigation panel.